model/clusters_mgmt/v1/add_on_installation_type.model

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright (c) 2019 Red Hat, Inc.
+Copyright (c) 2020 Red Hat, Inc.
```

Comment:
Undo this part, we currently only keep the copyright date of when the file was created